### PR TITLE
AEH/private_code_redesign 

### PR DESCRIPTION
**Pull request by aidan-hubley**
**Changes**
- Updated the visuals to the sharing private circle code view
- Deleted the img used to denote the hidden private code
- Added new icon to better represent visibility status 

**Notes**
- Using absolute positioning for some icons, need to be tested on other devices

**YouTrack Link**
- N/A

### CODE REVIEW

File: app/(settings)/shareCircle.js [hunk 1]
```diff
@@ -65,34 +72,65 @@ import QRCode from 'react-qr-code';
...
-										/>
 										<StyledText
-											className='px-[5px] font-bold text-3xl text-offblack text-center'
+											className='font-bold text-3xl text-offwhite'
 											onPress={() => shareCircle()}
 										>
 											0987654321
 										</StyledText>
+										<StyledView className='absolute right-3 top-3'>
+											<StyledIcon
```
Comment (dev-arekusandoru) [inline app/(settings)/shareCircle.js:91]: This icon would look better on the left side of the container

Comment (dev-arekusandoru) [reply app/(settings)/shareCircle.js:91]: it is also really close to the code when it is rendered, should have more spacing

File: app/(settings)/shareCircle.js [hunk 2]
```diff
@@ -65,34 +72,65 @@ import QRCode from 'react-qr-code';
...
 								)}
 							</StyledPressable>
 						</StyledView>
 					</StyledView>
 				</ScrollView>
+			<StyledView
+				style={{ top: Platform.OS == 'android' ? insets.top + 15 : 15 }}
+				className='absolute w-screen flex flex-row items-center justify-between px-[15px]'
+			>				
+				<StyledView></StyledView>
```
Comment (dev-arekusandoru) [inline app/(settings)/shareCircle.js]: what is this view for

File: app/(settings)/shareCircle.js [hunk 3]
```diff
@@ -65,34 +72,65 @@ import QRCode from 'react-qr-code';
...
+				<Button
```
Comment (dev-arekusandoru) [inline app/(settings)/shareCircle.js]: This button is a little too big, it should be made smaller

File: app/(settings)/shareCircle.js [hunk 4]
```diff
@@ -112,6 +150,76 @@ import QRCode from 'react-qr-code';
...
+							/>
+							<StyledText className='ml-3 text-2xl text-offwhite'>
+								Share the private code
+							</StyledText>
+							<StyledText className='absolute top-10 left-10 text-l text-offwhite'>
+								Only share with trusted people!
+							</StyledText>
+						</StyledView>					
+					</StyledView>
+					<StyledView className='items-center'>
```
Comment (dev-arekusandoru) [inline app/(settings)/shareCircle.js]: This button is really close to the bottom of the modal. It should be equal to the spacing from the top

**Review by dev-arekusandoru — CHANGES_REQUESTED** [review]
overall looks good, just a few visual changes needed

File: app/(settings)/shareCircle.js (hunk 2 above)
Comment (aidan-hubley) [reply app/(settings)/shareCircle.js]: This floats the button to the right... I'll find a better way to do this

File: app/(settings)/shareCircle.js (hunk 3 above)
Comment (aidan-hubley) [reply app/(settings)/shareCircle.js]: I disagree, its the same size as the other buttons on this page

**Review by dev-arekusandoru — APPROVED** [review]
looks good

**Pull request merged**